js/languages/en-US.json

```diff
@@ -155,7 +155,35 @@
     "editListingReturnText": "Return to listing",
     "freeShippingBanner": "Free Shipping",
     "listingDataChangedPopin": "Listing data has changed. %{refreshLink}",
-    "listingDataChangedPopinRefresh": "refresh"
+    "listingDataChangedPopinRefresh": "refresh",
+    "viewPhotos": "View %{count} Photos",
+    "type": "Type: %{type}",
+    "condition": "Condition: %{condition}",
+    "tags": "Tags",
+    "buyNow": "BuyNow",
+    "description": "Description",
+    "reviews": "Reviews",
+    "attachments": "Attachments",
+    "refundPolicy": "Refund Policy",
+    "termsAndConditions": "Terms Of Service",
+    "shipping": "Shipping",
+    "shipsFrom": "Ships from %{country}",
+    "shipTo": "Ship to:",
+    "shipToPlaceholder": "Select a Country",
+    "noShippingFound": "This listing does not ship to the selected country.",
+    "formats": {
```

Comment:
I was kind of torn on this too. I think in this case, since they are referring to labels that are repeated elsewhere, it would be ok to move them outside the sections and use a single set in multiple places.